src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
main() is not implemented
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/fd323257-2177-4fdb-a6b7-12b1c4205f2c)